authz/src/test/java/com/flipkart/varadhi/authz/DefaultAuthorizationProviderTest.java

```diff
@@ -197,28 +198,42 @@ public void testIsAuthorized_UserProjectAccess(VertxTestContext testContext) {
         Checkpoint checkpoint = testContext.checkpoint(3);
         defaultAuthorizationProvider
                 .init(authorizationOptions)
-                .compose(t -> defaultAuthorizationProvider.isAuthorized(TestUtil.testUser("proj_user2", false),
+                .compose(t -> defaultAuthorizationProvider.isAuthorized(testUser("proj_user2", false),
                         ResourceAction.PROJECT_GET, "flipkart/team_rocket/proj001"
                 )) // proj_user2 only has topic read access, so should fail
                 .onComplete(testContext.succeeding(t -> {
                     Assertions.assertFalse(t);
                     checkpoint.flag();
                 }))
 
-                .compose(t -> defaultAuthorizationProvider.isAuthorized(TestUtil.testUser("proj_user2", false),
+                .compose(t -> defaultAuthorizationProvider.isAuthorized(testUser("proj_user2", false),
                         ResourceAction.TOPIC_GET, "flipkart/team_rocket/proj001"
                 )) // proj_user2 only has topic read access, so should fail
                 .onComplete(testContext.succeeding(t -> {
                     Assertions.assertTrue(t);
                     checkpoint.flag();
                 }))
 
-                .compose(t -> defaultAuthorizationProvider.isAuthorized(TestUtil.testUser("proj_user1", false),
+                .compose(t -> defaultAuthorizationProvider.isAuthorized(testUser("proj_user1", false),
                         ResourceAction.PROJECT_GET, "flipkart/team_rocket/proj001"
                 )) // proj_user1 is project.read so should work
                 .onComplete(testContext.succeeding(t -> {
                     Assertions.assertTrue(t);
                     checkpoint.flag();
                 }));
     }
+
+    public static UserContext testUser(String name, boolean expired) {
```

Comment:
Might be better to put into testFixtures, can possibly used by other modules as well ?

Reply:
sure will do that

Reply:
done